libbeat/publisher/pipeline/module.go

```diff
@@ -116,17 +117,23 @@ func loadOutput(
 	beatInfo beat.Info,
 	monitors Monitors,
 	outcfg common.ConfigNamespace,
+	outRequired bool,
 ) (outputs.Group, error) {
 	log := monitors.Logger
 	if log == nil {
 		log = logp.L()
 	}
 
 	if publishDisabled {
+		log.Info("No outputs are configured, starting in paused mode.")
```

Comment:
This condition is not about 'outRequired', but we end here if publishing is disabled from the CLI, even if outputs are configured.

Reply:
:man_facepalming: I'll move it :innocent:  are we ok with the text istelf?